### ISSUE TITLE
Non-exsitant lightsaber being rendered in multiplayer

### ISSUE THREAD

**Issue by fr-Pursuit** [FiskFille/AdvancedLightsabers#56]
Hello,

While playing on a multiplayer server, a lightsaber appears on every distant player even when they don't have one.

This is due to a bug in ClientEventHandler.onRenderPlayerSpecialsPre()
Indeed, the condition `data != LightsaberData.EMPTY` should be replaced with `data.equals(LightsaberData.EMPTY)`, as the reference comparison you're using only works when `data` was set locally from the `LightsaberData.EMPTY` constant.

Regards,
Pursuit

**Comment by FiskFille** [comment]
Nice find!

**Comment by Roach9000** [comment]
Hey man where can i find that file so i can edit it out its annoying the fuck out of my friends and me?

**Comment by echsim** [comment]
Yes it would be very nice. the problem is that you even see this wenn youre invisible! i have no Idea where i could find that file

**Comment by AnrDaemon** [comment]
It's `src/main/java/com/fiskmods/lightsabers/common/event/ClientEventHandler.java`

It has a number of other issues, too.
